cpu/stm32_common/periph/pm.c

```diff
@@ -40,18 +42,71 @@
  *
  * Available values can be found in reference manual, PWR section, register CR.
  */
-#define PM_STOP_CONFIG (PWR_CR_LPDS | PWR_CR_FPDS)
+#if defined(CPU_FAM_STM32F0)
+#define PM_STOP_CONFIG  (PWR_CR_LPDS)
+#elif defined(CPU_FAM_STM32L0) || defined(CPU_FAM_STM32L1)
+#define PM_STOP_CONFIG  (PWR_CR_LPSDSR | PWR_CR_ULP)
+#else
+#define PM_STOP_CONFIG  (PWR_CR_LPDS | PWR_CR_FPDS)
+#endif
+#endif
+
+#ifndef PM_STOP_CLEAR
+/**
+ * @brief Define flags cleared before configuring stop mode
+ */
+#if defined(CPU_FAM_STM32L0) || defined(CPU_FAM_STM32L1)
+#define PM_STOP_CLEAR   (PWR_CR_LPSDSR | PWR_CR_PDDS)
+#else
+#define PM_STOP_CLEAR   (PWR_CR_LPDS | PWR_CR_PDDS)
+#endif
+#endif
+
+static inline uint32_t _ewup_config(void)
+{
+    uint32_t tmp = 0;
+#ifdef PM_EWUP_CONFIG
+    tmp |= PM_EWUP_CONFIG;
+#elif defined(PWR_CSR_EWUP)
+    tmp |= PWR_CSR_EWUP;
+#else
```

Comment:
I think this should be done differently.
In the current state, if the user doesn't provide a define for `PW_EWUP_CONFIG`, then this function will activate all wake-up pins.
This results in an invalid default STANDBY configuration for most of the nucleo boards in this case. For example, on nucleo-f091rc, with this default configuration, the STANDBY mode doesn't work unless one builds the application with `CFLAGS=-DPM_EWUP_CONFIG=0`.

What about having the possibility to provide the `PM_EWUP_CONFIG` in the board `periph_conf.h` and include this file in `pm.c` ?
Then there's no need for this function, just use the following snippet in the STANDBY case, below:
```c
PWR->CSR |= PM_EWUP_CONFIG
```
And provide a default PM_EWUP_CONFIG to 0 somewhere in the `periph_cpu_common.h` file.